degirum_tools/environment.py

```diff
@@ -60,6 +60,9 @@ def get_test_mode() -> bool:
 
 def get_token() -> str:
     """Returns a token from .env file"""
+    if in_colab():
+        from google.colab import userdata
+        return userdata.get('DEGIRUM_CLOUD_TOKEN')        

```

Comment:
Should we fallback to env.ini method if Colab secret is empty/not defiend?

Reply:
google colab returns a clear message saying that the secret is not defined. We cannot fall back to env.ini because that requires cloning the repo and having the file